### DISCUSSION TITLE
graphql/envelop/plugins/use-engine

### DISCUSSION THREAD

**Discussion by giscus** [dotansimha/envelop#1573] [Docs Discussions]
# graphql/envelop/plugins/use-engine

GraphQL Envelop: documentation

https://the-guild.dev/graphql/envelop/plugins/use-engine

**Comment by o-az** [comment]
Both of the following pages show the same thing:

1. https://the-guild.dev/graphql/envelop/plugins/use-schema-by-context
2. https://the-guild.dev/graphql/envelop/plugins/use-engine
